api-report/odsp-driver-web-cache.api.md

```diff
@@ -0,0 +1,38 @@
+## API Report File for "@fluidframework/odsp-driver-web-cache"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```

Comment:
is there anything ODSP specific for package to have ODSP in name?
I believe R11S is looking to have similar functionality and likely would love to reuse code if it fits their needs.
@znewton - FYI

Reply:
Persisted cache is a contract in ODSP driver, is that moving out?

Reply:
You can see all the imports we have from odsp-driver-definitions

Reply:
Got it. I'll leave it up to @znewton to comment here - I believe he was planning to make these interfaces generic (i.e., moving out of ODSP code base)

Reply:
But I can rename it driver-web-cache if we confident it eventually will have no ODSP contract dependencies? I have my doubts due to the "file" nature of this

Reply:
Ok, I'll rename it!

Reply:
@vladsud - should I wait for @znewton to chime in or just merge with the generic name and rename later if we can't align on the interfaces?

Reply:
I'd keep all the names generic and merge (even though interfaces are coming from ODSP). I think we will want to move them out of ODSP no matter what.